system_tests/monitoring.py

```diff
@@ -207,9 +207,14 @@ def _query_timeseries_with_retries():
             def _has_timeseries(result):
                 return len(list(result)) > 0
 
+            def _unknown_metric(result):
+                return ('The provided filter doesn\'t refer to any known '
+                        'metric.'in result.message)
```

Comment:
`message` is prose, so it may be subject to more change. See my comment https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2451#issuecomment-249969812 below